app/models/user.rb

```diff
@@ -292,8 +292,8 @@ def reassign_logs_and_update_organisation(new_organisation, log_reassignment)
     return unless new_organisation
 
     ActiveRecord::Base.transaction do
-      lettings_logs_to_reassign = assigned_to_lettings_logs
-      sales_logs_to_reassign = assigned_to_sales_logs
+      lettings_logs_to_reassign = assigned_to_lettings_logs.visible
+      sales_logs_to_reassign = assigned_to_sales_logs.visible
```

Comment:
Does this cause problems if a user has pending logs from a bulk upload that they may later choose to load?

Reply:
Yes, that's a very good point! I'll follow up what we want to do with those on the ticket